core/dbt/tests/fixtures/project.py

```diff
@@ -14,6 +15,13 @@
 
 # These are the fixtures that are used in dbt core functional tests
 
+# Logbook warnings are ignored so we don't have to fork logbook to support python 3.10.
+# This _only_ works for tests in `tests/` that use the project fixture.
+@pytest.fixture(scope="class")
```

Comment:
I don't think this needs to be a fixture because I can't conceive of a situation in which a test would want to turn the warning back on. So I think it would be better to just put it in the code, maybe right above setup_event_handler in the project fixture?